.pre-commit-config.yaml

```diff
@@ -7,7 +7,6 @@ repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v2.3.0
   hooks:
-  - id: check-added-large-files
```

Comment:
Why are we dropping this? You can [add a larger max size]
(https://github.com/pre-commit/pre-commit-hooks#check-added-large-files).

Reply:
> Let's get the updated build system in before we actually push 0.10.0, too.

Sure, no problem. The fact that I updated the CHANGELOG does not mean that I would push a release out. I remember you saying you have things in the pipeline. I just noticed the CHANGELOG was rather out-of-date.

Reply:
The file with all nuclei info is actually just over 500 KB, the limit. That line was not allowing me to commit :S. But it's most likely safe to re-add the line after this PR … (I had a quick look to see how to increase the limit to 1MB but did not immediately find a solution, hence went pragmatically as I just wanted to get the job done.)

Reply:
`args: ['--maxkb=600']` (placed just below the id)

Reply:
Done, thanks.